cai_causal_graph/graph_components.py

```diff
@@ -310,6 +325,45 @@ def __hash__(self) -> int:
         """Return a hash value of the node identifier."""
         return super().__hash__()
 
+    def to_dict(self, include_meta: bool = True) -> dict:
+        """
+        Return a dictionary representation of the time series node.
```

Comment:
Follow the example for `to_dict` for `Node` and `Edge` which includes full path to the classes.